src/routes/profile/[username]/+page.server.ts

```
@@ -233,23 +232,66 @@ export const actions: Actions = {
+
+		if (typeof name !== 'string' || !name.trim()) {
+			return fail(400, { message: 'Name is required' })
+		}
+
+		if (name.length > 50) {
+			return fail(400, { message: 'Name must be under 50 characters' })
+		}
+
+		const update_payload: Partial<typeof user.$inferSelect> = {
+			name: name.trim(),
+			bio: typeof bio === 'string' ? bio.trim() : null
```

ESLint

[error] Use undefined instead of null.

(no-restricted-syntax)